### DISCUSSION TITLE
3D Reconstruction of a moving object with static cameras

### DISCUSSION THREAD

**Discussion by luki446** [colmap/colmap#2425] [General]
Is this possible to run 3D reconstruction on moving object from photos gathered from 3 static cameras looking at pathway of this object?

**Comment by tmnku** [comment]
I guess you want to get a 3D model of the car? This would only be possible if you knew the poses of the car with respect to the cameras. Then you could create a dense reconstruction by making the car world fixed and placing the cameras relative to the car for stereo matching. However getting the cars poses will be the problem here.
